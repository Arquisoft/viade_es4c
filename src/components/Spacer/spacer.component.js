import React from "react";
import "./spacer.component.css";

const Spacer = (props) => {
	return (
		<div className="spacer">
			&nbsp;
		</div>
	);
};

export default Spacer;