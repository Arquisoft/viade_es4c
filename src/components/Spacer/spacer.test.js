import React from 'react';
import { shallow } from 'enzyme';
import Spacer from "./spacer.component";

describe('Spacer', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Spacer debug />);
  
    expect(component).toMatchSnapshot();
  });
});
