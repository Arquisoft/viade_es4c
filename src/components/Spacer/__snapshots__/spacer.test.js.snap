// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer should render correctly in "debug" mode 1`] = `
<div
  className="spacer"
>
  Â 
</div>
`;
